pyproject.toml

```diff
@@ -134,7 +134,7 @@ requests-kerberos = { version = "^0.14.0", optional = true }
 rich = "^13.3.4"
 sasl = { version = "^0.3.1", optional = true }
 smart-open = { version = "^6.3.0", extras = ["azure", "gcs"], optional = true }
-snowflake-connector-python = { version = "^3.7.1", optional = true }
+snowflake-connector-python = { version = "3.7.1", optional = true }  # Breaking ResultMetadata does not have fields
```

Comment:
@kchua78 I suspect the fixtures you added using the freeware including this commit (https://github.com/featurebyte/freeware/pull/95/files) but current `0.2.18` freeware version does not include that commit.